content/en/tracing/trace_collection/proxy_setup/_index.md

```diff
@@ -32,6 +32,9 @@ further_reading:
 - link: "https://kubernetes.github.io/ingress-nginx/user-guide/third-party-addons/opentelemetry/"
   tag: "Documentation"
   text: "OpenTelemetry for Ingress-NGINX Controller"
+- link: "https://github.com/DataDog/httpd-datadog"
```

Comment:
This repo is currently private

Reply:
That's on purpose. The code is not ready yet.